runtime/src/genesis_utils.rs

```diff
@@ -66,6 +67,7 @@ pub fn create_genesis_block_with_leader(
                 ),
             ),
         ])
+        // Bare minimum
```

Comment:
nit: use more words.  "Bare minimum ..... what"?

Reply:
sentence finished in https://github.com/solana-labs/solana/pull/5333